src/checkers/inference/model/Slot.java

```diff
@@ -37,7 +37,7 @@ public void setLocation(AnnotationLocation location) {
     public abstract Kind getKind();
 
     public enum Kind {
-        VARIABLE, CONSTANT, REFINEMENT_VARIABLE, EXISTENTIAL_VARIABLE, COMB_VARIABLE
+        VARIABLE, CONSTANT, REFINEMENT_VARIABLE, EXISTENTIAL_VARIABLE, COMB_VARIABLE, ARITHMETIC_VARIABLE
```

Comment:
Please refactor this to multiple lines.

There is no formatter will auto refactor this for you. See discussion here: https://github.com/typetools/annotation-tools/issues/36

Reply:
Will modify my change after merging #160